pkgs/applications/networking/p2p/stig/default.nix

```diff
@@ -44,6 +44,8 @@ python3Packages.buildPythonApplication rec {
     "--deselect=tests/client_test/ttypes_test.py::TestTimestamp::test_string__month_day_hour_minute_second"
```

Comment:
Could you check while at it if the other tests disabled should stay disabled? Please. For example there's a test disabled which a comment above it indicates it fails on Darwin, but it's disabled for linux as well.

Reply:
Test works for me on Linux, I've moved it to the Darwin list: https://github.com/NixOS/nixpkgs/pull/147833